WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -95,7 +95,7 @@ class LoginTests: XCTestCase {
     // Unified self hosted login/out
     // Replaces testSelfHostedUsernamePasswordLoginLogout
     func testSelfHostedLoginLogout() {
-        _ = PrologueScreen().selectSiteAddress()
+        PrologueScreen().selectSiteAddress()
```

Comment:
> Using '_' to ignore the result of a Void-returning function is redundant

This syntax is used to ignore the return value of a method if you want to call the method but don't care about the return value, which can happen for methods that perform a side-effect while also returning a value (which, by the way, is not a good pattern, either perform a side-effect or return a value, avoid doing both).

We ignore the return value with `_` because the compiler warns about unused assignments ("hey, you called `let carrots = pickCarrots(quantity: 3)` but never used your `carrots`").

But ☝️ the method at the end of this chain of calls returned `Void` (nothing) so there was nothing to ignore and the syntax was redundant.